### PR TITLE
Add our playground to the project again

### PR DESCRIPTION
**Pull request by yoichitgy**
This PR addresses Issue #475.

- Add `Sample-iOS.playground` to the project.
- Set `Build Active Scheme` of the playground to true to enable importing `Swinject` package as mentioned in [this post](https://wwdcbysundell.com/2020/importing-swift-packages-into-a-playground-in-xcode12/).

### CODE REVIEW

**Comment by yoichitgy** [comment]
@1ucas @mpdifran Would one of you review this PR? 🙏

File: Swinject.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -444,12 +445,21 @@
 			name = Products;
 			sourceTree = "<group>";
 		};
+		98D60BA9269AD802002FF5CF /* Playgrounds */ = {
+			isa = PBXGroup;
+			children = (
+				98D60BAA269AD812002FF5CF /* Sample-iOS.playground */,
```
Comment (mpdifran) [inline Swinject.xcodeproj/project.pbxproj:451]: I don't think you added this file to the PR!

**Comment by 1ucas** [comment]
Sure!

**Comment by 1ucas** [comment]
I sent a PR to your branch from my Fork adding the XcodeGen property so we don't lose consistency and the playground's reference the next time we generate the project. 

Here's the link: [XcodeGen Fix](https://github.com/yoichitgy/Swinject/pull/1)

**Comment by yoichitgy** [comment]
Thanks 1ucas! I merged your PR 👍 😃

**Review by 1ucas — APPROVED** [review]
Thanks! It seems good to me.

**Comment by yoichitgy** [comment]
Thank you for the review! Merged it😃

**Pull request merged**